vendor/vendorize

```diff
@@ -22,9 +22,12 @@ File.open(SETUP_PATH, 'w') do |setup_file|
         gem.name
       )
 
-      setup_file << "$LOAD_PATH.unshift(File.expand_path(%s, __FILE__))\n" % [
-        "../#{gem.name}-#{gem.version}/lib".inspect
-      ]
+      template = "$LOAD_PATH.unshift(File.expand_path(%s, __FILE__))\n"
+      gem.require_paths.each do |path|
+        setup_file << template % [
```

Comment:
If you do `setup_file.puts` instead of `setup_file <<`, would that get rid of the `\n` in the template?